docs/source/amp.rst

```diff
@@ -1,19 +1,19 @@
 .. role:: hidden
     :class: hidden-section
 
-Automatic Mixed Precision package - torch.cuda.amp
+Automatic Mixed Precision package - torch.autocast
 ==================================================
 
-.. automodule:: torch.cuda.amp
-.. currentmodule:: torch.cuda.amp
+.. automodule:: torch.autocast
+.. currentmodule:: torch.autocast
 
-:class:`torch.cuda.amp` and :class:`torch` provide convenience methods for mixed precision,
+:class:`torch.autocast` and :class:`torch` provide convenience methods for mixed precision,
 where some operations use the ``torch.float32`` (``float``) datatype and other operations
-use ``torch.float16`` (``half``). Some ops, like linear layers and convolutions,
-are much faster in ``float16``. Other ops, like reductions, often require the dynamic
+use lower precision floating point (``lower_precision_fp``)：``torch.float16`` (``half``) or ``torch.bfloat16``. Some ops, like linear layers and convolutions,
+are much faster in ``lower_precision_fp``. Other ops, like reductions, often require the dynamic
 range of ``float32``.  Mixed precision tries to match each op to its appropriate datatype.
 
-Ordinarily, "automatic mixed precision training" uses :class:`torch.autocast` and
+Ordinarily, "automatic mixed precision training" on CUDA uses :class:`torch.autocast` and
 :class:`torch.cuda.amp.GradScaler` together, as shown in the :ref:`Automatic Mixed Precision examples<amp-examples>`
```

Comment:
It is also about `float16` training. Bfloat16 doesn't need `GradScaler`. Suggest to clarify this.

Reply:
Thank you. After a second thought, I have put `torch.cpu.amp.autocast` into a separate file `cpu.amp.rst`. Then we don't need to mention `GradScaler` in that file. Now:
* `amp.rst` is the entry point for `torch.autocast`.
* `cuda.amp.rst` describes the autocast behavior on CUDA `torch.cuda.amp.autocast`.
*  `cpu.amp.rst` describes the autocast behavior on CPU `torch.cpu.amp.autocast`.